### PR TITLE
Add "Last Updated" To The Website

### PR DESCRIPTION
**Pull request by PierrickGT**
## Description

This PR adds the current updated page date to the Layout and  the last updated page date to the Footer.

## Related Issue

https://github.com/ethereum/ethereum-org-website/issues/49

## Screenshots (if appropriate):

Screenshot 2019-10-28 at 19 34 41
Screenshot 2019-10-28 at 22 31 53

## Checklist:

- [x] I have read the contributing guidelines in the project README.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by samajammin** [comment]
Thanks for your submission! Will review ASAP - definitely by end of this week.

File: docs/.vuepress/theme/Layout.vue [hunk 1]
```diff
@@ -82,6 +85,12 @@
         },
         userPageClass
         ]
+      },
+      lastUpdatedDate() {
+          const moment = require('moment')
+
+          moment.locale(this.$page.lang)
```
Comment (samajammin) [inline docs/.vuepress/theme/Layout.vue]: ❤️

File: docs/.vuepress/theme/components/Footer.vue [hunk 2]
```diff
@@ -17,6 +18,29 @@
   </footer>
 </template>
 
+<script>
+  export default {
+    computed: {
+      lastUpdatedDate() {
+        const moment = require('moment')
```
Comment (samajammin) [inline docs/.vuepress/theme/components/Footer.vue]: Let's move this require statement outside the computed property. Remove this line. Just below the `<script>` tag on line 21, you should be able to add a line: ```import moment from 'moment';```

File: docs/.vuepress/theme/components/Footer.vue [hunk 3]
```diff
@@ -17,6 +18,29 @@
...
+
+        function compareTimestamps(a, b) {
```
Comment (samajammin) [inline docs/.vuepress/theme/components/Footer.vue]: Let's avoid declaring this function every time in the computed property. Below this move this to the `methods` object of your component:
```
    methods: {
      compareTimestamps: function(a, b) {
        if (a.lastUpdated < b.lastUpdated)
          return -1;
        if (a.lastUpdated > b.lastUpdated)
          return 1;
        return 0;
      }
    },
```
Then from this computed property, you can call `this.compareTimestamps`

File: docs/.vuepress/theme/components/Footer.vue [hunk 4]
```diff
@@ -41,6 +65,9 @@
       div.credits
         display block
 
+    p.updated-date
+      margin 1em 0
```
Comment (samajammin) [inline docs/.vuepress/theme/components/Footer.vue]: Let's also add `color $subduedColor`

File: docs/.vuepress/theme/Layout.vue [hunk 5]
```diff
@@ -2,7 +2,10 @@
   <div id="wrapper" :class="pageClasses">
     <Header :class="{ 'home': isLanding }" @toggle-sidebar="toggleSidebar" @toggle-mode="toggleMode" />
     <Hero v-if="isLanding" :dark="darkMode" />
-    <main :class="contentClasses"><Content/></main>
+    <main :class="contentClasses">
+      <p v-if="!isLanding">Page last updated: {{lastUpdatedDate}}</p>
```
Comment (samajammin) [inline docs/.vuepress/theme/Layout.vue]: Please add a class & style to this `<p>` that applies: `color $subduedColor`

File: docs/.vuepress/theme/components/Footer.vue (hunk 3 above)
Comment (PierrickGT) [reply docs/.vuepress/theme/components/Footer.vue]: Oh yeah thanks, I'm pretty new to Vue.js and didn't think about this.
The changes you've asked for are in this commit: https://github.com/ethereum/ethereum-org-website/pull/356/commits/483f17d81c53bc800a9f039c7f5e89daf1c056f7

**Comment by PierrickGT** [comment]
Thanks @samajammin! 
Hope I'll be able to continue to contribute too!

**Pull request merged**